lua/entities/gmod_wire_expression2/core/extpp.lua

```diff
@@ -80,7 +80,7 @@ local function parseParameters(raw, trace)
 			return parsed, true, name
 		elseif raw_param:match("^%s*%.%.%.%s*$") then -- Variadic lua parameter
 			assert(k == len, "PP syntax error: Ellipses (...) must be the last argument.")
-			ErrorNoHalt("Warning: Use of variadic parameter with ExtPP is not recommended and deprecated. Instead use ...<name> (which passes a table) or the `args` variable " .. trace .. "\n")
+			WireLib.NotifyAll("Use of variadic parameter with ExtPP is not recommended and deprecated. Instead use ...<name> (which passes a table) or the `args` variable " .. trace .. "\n", 2)
```

Comment:
Is this an E2 warning? I would get rid of this in favor of E2's own stuff. This should be notifying the player trying to use it.

Reply:
It's a preprocessor warning from an addon using `e2function` wrong.